Source/ISHRemote/Trisoft.ISHRemote/Cmdlets/TrisoftCmdlet.cs

```diff
@@ -18,14 +18,12 @@
 using System.Collections.Generic;
 using System.Linq;
 using System.Management.Automation;
-using Trisoft.ISHRemote.Objects;
-using Trisoft.ISHRemote.Objects.Public;
-using Trisoft.ISHRemote.Exceptions;
 using System.Threading;
 using System.Xml;
-using Trisoft.ISHRemote.Interfaces;
 using Trisoft.ISHRemote.Folder25ServiceReference;
-using System.Runtime.InteropServices;
+using Trisoft.ISHRemote.Interfaces;
```

Comment:
Surprised why removing a function, which I assume was not used (`DevideListInBatchesByLogicalId`) results in 3 extra using statements

Reply:
It's not extra, the using section is ordered